Hum parace que realmente tem um erro agora com o que entrou na 16.0 nessas últimas horas: 

```
2026-07-31 19:35:34,873 646 INFO odoo odoo.addons.mail.models.mail_mail: Sent batch 1 emails via mail server ID #False 
2026-07-31 19:35:34,999 646 INFO odoo odoo.addons.l10n_br_sale_stock.tests.test_sale_stock: ====================================================================== 
2026-07-31 19:35:34,999 646 ERROR odoo odoo.addons.l10n_br_sale_stock.tests.test_sale_stock: ERROR: TestSaleStock.test_picking_sale_order_product_and_service
Traceback (most recent call last):
  File "/__w/l10n-brazil/l10n-brazil/l10n_br_sale_stock/tests/test_sale_stock.py", line 163, in test_picking_sale_order_product_and_service
    sale_order._create_invoices(final=True)
  File "/__w/l10n-brazil/l10n-brazil/l10n_br_sale/models/sale_order.py", line 139, in _create_invoices
    moves |= super()._create_invoices(
  File "/opt/odoo-venv/lib/python3.10/site-packages/odoo/addons/sale_invoice_plan/models/sale.py", line 136, in _create_invoices
    moves = super()._create_invoices(grouped=grouped, final=final, date=date)
  File "/opt/odoo/addons/sale/models/sale_order.py", line 1211, in _create_invoices
    moves = self.env['account.move'].sudo().with_context(default_move_type='out_invoice').create(invoice_vals_list)
  File "<decorator-gen-276>", line 2, in create
  File "/opt/odoo/odoo/api.py", line 431, in _model_create_multi
    return create(self, arg)
  File "/__w/l10n-brazil/l10n-brazil/l10n_br_account/models/account_move.py", line 119, in create
    return super().create(vals_list)
  File "<decorator-gen-237>", line 2, in create
  File "/opt/odoo/odoo/api.py", line 431, in _model_create_multi
    return create(self, arg)
  File "/opt/odoo-venv/lib/python3.10/site-packages/odoo/addons/account_payment_partner/models/account_move.py", line 183, in create
    return super().create(vals_list)
  File "<decorator-gen-232>", line 2, in create
  File "/opt/odoo/odoo/api.py", line 431, in _model_create_multi
    return create(self, arg)
  File "/opt/odoo/addons/purchase/models/account_invoice.py", line 134, in create
    moves = super(AccountMove, self).create(vals_list)
  File "<decorator-gen-212>", line 2, in create
  File "/opt/odoo/odoo/api.py", line 431, in _model_create_multi
    return create(self, arg)
  File "/opt/odoo/addons/account/models/account_move.py", line 2313, in create
    with self._check_balanced(container):
  File "/usr/lib/python3.10/contextlib.py", line 142, in __exit__
    next(self.gen)
  File "/opt/odoo/addons/account/models/account_move.py", line 1774, in _check_balanced
    raise UserError(error_msg)
odoo.exceptions.UserError: An error has occurred.

The move (Draft Invoice (* 363) (Customer Ref Test 2)) is not balanced
```

--- a/l10n_br_fiscal/models/tax.py
+++ b/l10n_br_fiscal/models/tax.py
@@ -692,7 +692,8 @@ class Tax(models.Model):
         tax_dict_pis = taxes_dict.get("pis", {})
         tax_dict_cofins = taxes_dict.get("cofins", {})
         if (
-            cfop.destination == CFOP_DESTINATION_EXPORT
+            cfop
+            and cfop.destination == CFOP_DESTINATION_EXPORT
             and fiscal_operation_type == FISCAL_IN
         ):
             tax_dict["add_to_base"] += (
@@ -725,7 +726,8 @@ class Tax(models.Model):
         tax_dict_pis = taxes_dict.get("pis", {})
         tax_dict_cofins = taxes_dict.get("cofins", {})
         if (
-            cfop.destination == CFOP_DESTINATION_EXPORT
+            cfop
+            and cfop.destination == CFOP_DESTINATION_EXPORT
             and fiscal_operation_type == FISCAL_IN
         ):
             tax_dict["add_to_base"] += (
@@ -756,7 +758,8 @@ class Tax(models.Model):
         tax_dict_pis = taxes_dict.get("pis", {})
         tax_dict_cofins = taxes_dict.get("cofins", {})
         if (
-            cfop.destination == CFOP_DESTINATION_EXPORT
+            cfop
+            and cfop.destination == CFOP_DESTINATION_EXPORT
             and fiscal_operation_type == FISCAL_IN
         ):
             tax_dict["add_to_base"] += tax_dict_ii.get("tax_value", 0.00) + kwargs.get(

--- a/l10n_br_fiscal/models/tax.py
+++ b/l10n_br_fiscal/models/tax.py
@@ -688,6 +688,7 @@ class Tax(models.Model):
         tax_dict_ii = taxes_dict.get("ii", {})
         tax_dict_is = taxes_dict.get("is", {})
         tax_dict_icms = taxes_dict.get("icms", {})
+        tax_dict_issqn = taxes_dict.get("issqn", {})
         tax_dict_pis = taxes_dict.get("pis", {})
         tax_dict_cofins = taxes_dict.get("cofins", {})
         if (
@@ -702,6 +703,7 @@ class Tax(models.Model):
         else:
             tax_dict["remove_from_base"] += (
                 tax_dict_icms.get("tax_value", 0.00)
+                + tax_dict_issqn.get("tax_value", 0.00)
                 + tax_dict_pis.get("tax_value", 0.00)
                 + tax_dict_cofins.get("tax_value", 0.00)
             )
@@ -719,6 +721,7 @@ class Tax(models.Model):
         tax_dict_ii = taxes_dict.get("ii", {})
         tax_dict_is = taxes_dict.get("is", {})
         tax_dict_icms = taxes_dict.get("icms", {})
+        tax_dict_issqn = taxes_dict.get("issqn", {})
         tax_dict_pis = taxes_dict.get("pis", {})
         tax_dict_cofins = taxes_dict.get("cofins", {})
         if (
@@ -733,6 +736,7 @@ class Tax(models.Model):
         else:
             tax_dict["remove_from_base"] += (
                 tax_dict_icms.get("tax_value", 0.00)
+                + tax_dict_issqn.get("tax_value", 0.00)
                 + tax_dict_pis.get("tax_value", 0.00)
                 + tax_dict_cofins.get("tax_value", 0.00)
             )

